index.html

```diff
@@ -56,7 +58,6 @@ <h2 id="story">My Earlier Years</h2>
         width="600" height="450" frameborder="3" style="border:3"
         allowfullscreen align="middle"></iframe> </div> <br/>
 
```

Comment:
I am having trouble grabbing the lines of code, but if you go back to lines 35-48, it looks like you are using 4 spaces of white space instead of two.